pkg/clients/imagebuilder/client_test.go

```diff
@@ -830,6 +830,82 @@ var _ = Describe("Image Builder Client Test", func() {
 		Expect(img.Commit.ExternalURL).To(BeFalse())
 	})
 
+	It("test compose installer with activation key", func() {
+		orgID := "234"
+		installer := models.Installer{Username: faker.Username(), SSHKey: faker.UUIDHyphenated()}
+		composeJobID := "compose-job-id-returned-from-image-builder"
+		ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			b, err := io.ReadAll(r.Body)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(b).ToNot(BeNil())
+			var req ComposeRequest
+			err = json.Unmarshal(b, &req)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(req.Customizations.Subscription.ActivationKey).To(Equal("test-key"))
+			Expect(req.Customizations.Subscription.Organization).To(Equal(234))
+			Expect(req.Customizations.Subscription.RHC).To(Equal(true))
+			Expect(req.Customizations.Subscription.Insights).To(Equal(true))
+			Expect(req.Customizations.Subscription.BaseUrl).To(Equal(config.Get().SubscriptionBaseUrl))
+			Expect(req.Customizations.Subscription.ServerUrl).To(Equal(config.Get().SubscriptionBaseUrl))
```

Comment:
```suggestion
			Expect(req.Customizations.Subscription.ServerUrl).To(Equal(config.Get().SubscriptionServerURL))
```
Think also we should initialize these config values at the top of this test, and restore them in AfterEach function